cli/src/main.rs

```diff
@@ -194,8 +194,14 @@ fn main() {
                     .unwrap_or(false);
 
                     let plain: Arc<
-                        block::PlainBdev<hw::virtio::block::Request>,
-                    > = block::PlainBdev::create(disk_path, readonly).unwrap();
+                        block::PlainBdev<
```

Comment:
Nit: Don't need to explicitly specify the type here. I know this is existing but given you also added a new type parameter here, I figure might as well clean it up. Should be the same as line 252 below.

Reply:
:+1:, fixed in f5eccea